javascript/javascript2/week1/JS2W1.js

```diff
@@ -0,0 +1,82 @@
+////////////////////////////////////////////////////////////Find the shortest word
+
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+const number = [];
+
+function findTheShortest(words) {
+    for (let i = 0; i < words.length; i++) {
+        number.push(danishWords[i].length);
+    }
+    let index = number.indexOf(Math.min.apply(Math, number));
+    return words[index];
+};
+
+console.log(findTheShortest(danishWords));
+
+///////////////////////////////////////////////////////////////////////Find and count the Danish letters
+
+function findDanishLetters(danish) {
+    let total = 0;
+    let å = 0;
+    let æ = 0;
+    let ø = 0;
+
+    for (let i = 0; i < danish.length; i++) { //Search inside the string sentnecse 
+        let word = danish.substring(i, i + 1);
+        if (word === 'å') {
+            å++;
+        } else if (word === 'æ') {
+            æ++;
+        } else if (word === 'ø') {
+            ø++;
+        }
+    };
+    console.log(`total: ${å+æ+ø}, å:${å}, ø:${ø}, æ:${æ}`);
+};
```

Comment:
Try to name the variables properly, in this case you're not looking at words but at letters. 

Also, how about initializing the letters in an object in an object as the homework task specified
```
const danishLetters = {
  "å": 0, 
  "æ": 0,
  "ø": 0
}
```
and then adding 1 to the value when the key is found in the string?
You are also not counting the uppercase letters.

Also, it is preferable to `return` something (in this case the object above) from a function instead of logging out a result and making it return `undefined`. You can always log out what it returns to you, but try not to hardcode it inside the function.